### ISSUE TITLE
Password characters should be shown only upon request

### ISSUE THREAD

**Issue by rkaletka** [jpochyla/psst#169]
I don't even ask about how the password is being stored, but at least it could have not be shown as actual password's characters.
Anyways, it wouldn't be a big problem if I didn't check account preferences after switch to dark mode - there I found my login in password in plain (only after first launch).
Would label as minor bug + enhancement (asterisks on input)
Problem observed on Windows (don't know about others). Good job overall tho.

**Comment by leokhachatorians** [comment]
This is something I was interested in fixing myself, but it doesn't seem like this is supported in Druid quite yet: https://github.com/jpochyla/psst/issues/45

**Comment by jpochyla** [comment]
Duplicate of #45
